corehq/apps/accounting/tasks.py

```diff
@@ -618,3 +620,68 @@ def assign_explicit_community_subscription(domain_name, start_date):
         internal_change=True,
         service_type=SubscriptionType.PRODUCT,
     )
+
+@periodic_task(run_every=crontab(minute=0, hour=9), queue='background_queue', acks_late=True)
+def send_overdue_reminders():
+    today = datetime.date.today()
+    invoices = Invoice.objects.filter(is_hidden=False,
+                                      subscription__service_type=SubscriptionType.PRODUCT,
+                                      date_paid__isnull=True,
+                                      date_due__lt=today)\
+        .exclude(subscription__plan_version__plan__edition=SoftwarePlanEdition.ENTERPRISE)\
+        .order_by('date_due')\
+        .select_related('subscription__subscriber')
+
+    accounts = set()
+    for invoice in invoices:
+        if invoice.get_domain() not in accounts:
+            accounts.add(invoice.get_domain())
+            total = Invoice.objects.filter(
+                is_hidden=False,
+                subscription__subscriber__domain=invoice.get_domain()
+            ).aggregate(Sum('balance'))['balance__sum']
+            if total >= 100:
+                _apply_notices(invoice, today, total)
+
+def _apply_notices(invoice, today, total):
+    from corehq.apps.domain.views import DomainSubscriptionView
+    from corehq.apps.domain.views import DomainBillingStatementsView
+
+    days_ago = (today - invoice.date_due).days
+    context = {
+        'invoice': invoice,
+        'total': total,
+        'subscription_url': absolute_reverse(DomainSubscriptionView.urlname, args=[invoice.get_domain()]),
+        'statements_url': absolute_reverse(DomainBillingStatementsView.urlname, args=[invoice.get_domain()]),
+        'date_60': invoice.date_due + datetime.timedelta(days=60)
+    }
+    if days_ago == 61:
```

Comment:
(nit) Might be easier to follow if you called a separate function for each branch.

So,
if days_ago == 61:
    func1(...)
elif days_ago == 58:
    ...

Moving this branching to send_overdue_reminders also makes it clearer that when days_ago is not equal to 1, 58, or 61, you do nothing.  I might expect _apply_notices to do something, but it's not obvious that it's a noop.